### ISSUE TITLE
Temps keep rising

### ISSUE THREAD

**Issue by codingForCats** [trexminer/T-Rex#443]
Using 3060ti and can't remember when, but at some point my temperature started getting out of control.

Originally it was stable around 56-57 degrees, occasionally getting up to 58-59.

Now as soon as I start the miner, in less than 2 minutes it's up to 60 and keeps going higher. 

![Overheating](https://user-images.githubusercontent.com/13934303/115593336-3cd8ea00-a289-11eb-9962-7debdf230654.PNG)

**Comment by pdieppa** [comment]
From the manual:

--fan                      Sets GPU fan speed in percent or target temperature (auto-fan).
                                   Valid formats:
                                      --fan N   (where N is the fan speed)
                                      --fan t:N (where N is the target temperature)
                                   Example: --fan 45,t:67
                                      GPU #0: set fan speed to 45%
                                      GPU #1: maintain GPU core temperature at 67C

**Comment by codingForCats** [comment]
I'm using MSI afterburner and the fan speed curve (about 50% at 57degrees) was keeping it stable at 56-57 degrees. After the temps started going up, I upped the fan curve and the temps are still going up. Why set the fan speed from trex instead?

**Comment by pdieppa** [comment]
In my opinion, it's usually better to control GPU, memory and fan right from the miner since you don't need anything else running.
Example, if for whatever reason MSI Afterburner crashes, the card may revert back to the default settings causing the card to heat up and drawing higher power.

**Comment by pdieppa** [comment]
From the image you posted, you could lower the power usage by almost 70 watts.
Most of my 3060 Ti run at 119 watts and hashing at a little over 60mh. I don't run windows so I could not give you MSI overclock settings.

**Comment by codingForCats** [comment]
Thanks, yeah i was trying to be conservative with my settings because I use my computer all day. I'll give the miner settings a try and see if it helps

**Comment by deslumoprime** [comment]
Your cards should run below 130W. Turn the fans up, if they have proper access to fresh air, 70% fans WILL keep them under 60C. I also suggest removing the temp limit as you shouldn't need one if your cards are cooled properly. 

I suggest not modifying the card settings in TRex. Use MSI Afterburner. Set the power in MSI AB lower, somewhere in the 52-65 range depending on your model of card. Also set a static fan profile, maybe 70 or 80, 60 if you value your sanity, or 100 if you are deaf. Drop the core clock and moon the memory clock, I find that 1290 gets the same perf as 1300 but WAY less errors.

**Comment by DeadfoolBruh** [comment]
> From the manual:
> 
> --fan Sets GPU fan speed in percent or target temperature (auto-fan).
> Valid formats:
> --fan N (where N is the fan speed)
> --fan t:N (where N is the target temperature)
> Example: --fan 45,t:67
> GPU #0: set fan speed to 45%
> GPU #1: maintain GPU core temperature at 67C

Sorry about the necropost, I'm very new to this entire thing but where do i insert the commands? I can't type in the console and the only place I could think of is the .bat file.

**Comment by algavi555** [comment]
yes, you can add record "--fan t:62[30-100]" in your miner bat file in any place after "t-rex.exe"
that means hold 62 degrees on gpu with fan limits 30-100 percents
sorry for necroposting=)
